### PR TITLE
feat(pack.resiliation): add new reasons for resiliation survey

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-439
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

For the pack resiliation survey, add new reasons to justify the resiliation and a subsidiary question to give more detail for those reasons. Remove reasons not used yet.

Removed reasons :
- billingProblems,
- ftth,
- changeOfTerms,
- goToCompetitor

Added reasons :
- eligibilityFtth,
- price,
- changeOperator

Translated labels for removed reasons were removed too.

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.html [hunk 1]
```diff
@@ -130,11 +130,29 @@ <h1 data-translate="pack_resiliation_title"></h1>
...
                         </select>
                     </div>
 
+                    <div
+                        class="form-group mt-3"
+                        data-ng-if="PackResiliation.isSubsidiaryQuestion"
+                    >
+                        <label
+                            class="control-label"
+                            data-ng-bind="'pack_resiliation_susbsidiary_question_'  + PackResiliation.model.reason.value | translate"
```
Comment (JeremyDec) [inline packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.html:145]:
```suggestion
                            data-translate="{{ 'pack_resiliation_susbsidiary_question_'  + PackResiliation.model.reason.value }}"
```

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js [hunk 2]
```diff
@@ -4,200 +4,215 @@ import remove from 'lodash/remove';
...
-
-  this.init = function init() {
-    self.subServicesTerms = null;
-    self.subServicesTermsLoading = false;
-    self.loading = true;
-    self.model.when = null;
-    self.dpOpts = {};
-
-    OvhApiPackXdslResiliation.Aapi()
+export default class PackResiliationCtrl {
```
Comment (JeremyDec) [inline packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:7]: 👍

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js [hunk 3]
```diff
@@ -398,8 +419,23 @@ export default /* @ngInject */ function PackResiliationCtrl(
...
+            this.resiliatePack(this);
+            break;
+        }
+        return result;
+      });
+  }
+
+  checkNeedSubsidiaryQuestion() {
+    this.isSubsidiaryQuestion =
+      ['technicalProblems', 'eligibilityFtth', 'changeOperator'].indexOf(
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:437]: **nitpick:** We can use `Array.prototype.includes`.

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js [hunk 4]
```diff
@@ -206,163 +221,168 @@ export default /* @ngInject */ function PackResiliationCtrl(
...
 
   /**
    * Resiliate a pack
    * @param  {Object} pack   Pack to resiliate
    * @param  {Object} survey Reason to resiliate
    * @param {Boolean} accept If true the resiliation must be done
    */
-  this.resiliatePack = function resiliatePack() {
+  resiliatePack(slf) {
+    const self = slf;
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:340]: **question:** Why is this required?

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js [hunk 5]
```diff
@@ -4,200 +4,215 @@ import remove from 'lodash/remove';
...
     );
-  };
+  }
+
+  /**
+   * Get the current currency symbol
+   *
+   * @return promise with the symbol of the current currency
+   */
+  getCurrentCurrencySymbol() {
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:159]: **question:** This method still need to be asynchronous?

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js (hunk 4 above)
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:340]: I don't know why but it works like that. If I don't those 2 different values, it doesn't work. If you have a better practice, I'll take it.

File: packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js (hunk 5 above)
Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js:159]: Yes

**Review by antleblanc — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

File: yarn.lock [hunk 6]
```diff
@@ -4420,13 +4420,111 @@
   resolved "https://registry.yarnpkg.com/@open-draft/until/-/until-1.0.3.tgz#db9cc719191a62e7d9200f6e7bab21c5b848adca"
   integrity sha512-Aq58f5HiWdyDlFffbbSjAlv596h/cOnt2DO1w3DOC7OJ5EHs0hd/nycJfiu9RJbT6Yk6F1knnRRXNSpxoIVZ9Q==
 
-"@ovh-ux/ng-ovh-api-wrappers@^4.0.11", "@ovh-ux/ng-ovh-api-wrappers@^4.0.4":
+"@ovh-ux/component-rollup-config@^11.0.1":
```
Comment (antleblanc) [inline yarn.lock:4423]: Following diff has been fixed via #6772.

Before merging this Pull Request, a rebase will be required.

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: yarn.lock (hunk 6 above)
Comment (antleblanc) [reply yarn.lock:4423]: Rebasing on `develop` branch in order to remove the actual diff on `yarn.lock` file.

Comment (antleblanc) [reply yarn.lock]: Rebased successfully.

**Review by antleblanc — APPROVED** [review]
